### PR TITLE
renamed sample program to from helloserver to podtatoserver

### PR DESCRIPTION
**Pull request by AloisReitbauer**
These are the first changes to update the sample

### CODE REVIEW

**Review by thschue — APPROVED** [review]
lgtm

File: delivery/CNABwithPorter/porter.yaml [hunk 1]
```diff
@@ -3,10 +3,10 @@
 # See https://porter.sh/author-bundles for documentation on how to configure your bundle
 # Uncomment out the sections below to take full advantage of what Porter can do!
 
-name: hello-server
+name: podtatoserver
 version: 0.1.0
 description: "A porter file for the hello server"
```
Comment (jetzlstorfer) [inline delivery/CNABwithPorter/porter.yaml:8]: should be renamed to "podtato server"?

**Review by jetzlstorfer — CHANGES_REQUESTED** [review]
found one occurrence of old name which should be changed

**Comment by yogeek** [comment]
In `delivery/manifest/manifest.yaml` you missed the names of the resources I think

**Pull request merged**